### DISCUSSION TITLE
Failed to Resolve Tags on Server.

### DISCUSSION THREAD

**Discussion by maxhunter2011** [shedaniel/RoughlyEnoughItems#1051] [Q & A]
Hi, been using REI on a server I'm playing on and loving it. When I click recipe for anything that's out in the wild (Ores, flowers, etc.) I get this pop up and no actual data on it. How can I fix this? Thank you!
![image](https://user-images.githubusercontent.com/37778579/182409622-ccd42f1d-58b2-4e91-9ea8-a2d52b066caa.png)

**Comment by gmanc2** [comment]
Same issue with me on fabric 1.19.2.

**Comment by electroninja08** [comment]
bump, this still isn’t fixed

**Comment by Bamberghh** [comment]
Still isn't fixed, NeoForge 1.21.1
